.semaphore/semaphore.yml.d/blocks/20-felix.yml

```diff
@@ -197,6 +197,16 @@
           - make check-wireguard
           - ../.semaphore/run-and-monitor fv-${SEMAPHORE_JOB_INDEX}.log make fv-no-prereqs FV_BATCHES_TO_RUN="${SEMAPHORE_JOB_INDEX}" FV_NUM_BATCHES=${SEMAPHORE_JOB_COUNT}
         parallelism: 3
+      - name: nftables FV Test matrix
+        execution_time_limit:
+          minutes: 120
+        env_vars:
+        - name: FELIX_FV_NFTABLES
+          value: "true"
```

Comment:
Should this be `Enabled` now?

Also noticed that Semaphore test results only seemed to have one set of results from the FVs.

What about BPF mode on an nftables system?  Do we need some tests for that?